executor/show_placement.go

```diff
@@ -387,3 +418,68 @@ func (e *ShowExec) getPolicyPlacement(policyRef *model.PolicyRefInfo) (settings
 	}
 	return policy.PlacementSettings, nil
 }
+
+func fetchScheduled(ctx context.Context, scheduleState map[int64]bool, id int64) (bool, error) {
+	if s, ok := scheduleState[id]; ok {
+		return s, nil
+	}
+	startKey := codec.EncodeBytes(nil, tablecodec.GenTablePrefix(id))
+	endKey := codec.EncodeBytes(nil, tablecodec.GenTablePrefix(id+1))
+	schedule, err := infosync.GetReplicationState(ctx, startKey, endKey)
```

Comment:
is there a batch interface?  one request for a single seems costy

Reply:
given the emergency, let merge it first

Reply:
No, there is no batch api.